### ISSUE TITLE
VS Code: vsix is missing quick-lint-js .wasm

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#356]

**Comment by strager** [comment]
.vsix file sizes:

Commit 9202bef5125b0349bf1539f674f179c7877bb741: 151'207 bytes
Commit 11ce52309942d495e71c6e1500f12bfe7e6f4ee7: 18'038 bytes

**Comment by strager** [comment]
It looks like upstream vsce changed, not quick-lint-js:

```
$ npx vsce@1.91.0 package --baseImagesUrl https://raw.githubusercontent.com/quick-lint/quick-lint-js/master/plugin/vscode/
npx: installed 72 in 3.557s
 INFO  Detected presence of yarn.lock. Using 'yarn' instead of 'npm' (to override this pass '--no-yarn' on the command line).
 DONE  Packaged: /home/strager/Projects/quicklint-js/plugin/vscode/quick-lint-js-0.2.0.vsix (9 files, 119.65KB)

$ npx vsce@1.92.0 package --baseImagesUrl https://raw.githubusercontent.com/quick-lint/quick-lint-js/master/plugin/vscode/
npx: installed 73 in 3.278s
 INFO  Detected presence of yarn.lock. Using 'yarn' instead of 'npm' (to override this pass '--no-yarn' on the command line).
 DONE  Packaged: /home/strager/Projects/quicklint-js/plugin/vscode/quick-lint-js-0.2.0.vsix (6 files, 17.62KB)
```

**Comment by strager** [comment]
https://github.com/microsoft/vscode-vsce/issues/576

**Comment by strager** [comment]
Fixed in commit 9031da153f581f146824d2f884b19ff5ff13f26f. Future issues mitigated in commit b55b6b5374b0547e28d034d012a4a0e3aff8de65.
